editor/src/components/inspector/nine-block-controls.tsx

```diff
@@ -112,13 +112,18 @@ const isSelectedSelector = createSelector(
   justifyAlignSelector,
   flexDirectionSelector,
   (_: MetadataSubstate, x: JustifyContentFlexAlignemt) => x,
-  (detectedJustifyContentFlexAlignment, flexDirection, fixedJustifyContentFlexAlignment) =>
-    detectedJustifyContentFlexAlignment != null &&
-    justifyContentAlignItemsEquals(
+  (detectedJustifyContentFlexAlignment, flexDirection, fixedJustifyContentFlexAlignment) => {
+    const defaultJustifyContentFlexAlignment: JustifyContentFlexAlignemt = {
+      justifyContent: 'flex-start',
+      alignItems: 'flex-start',
+    }
```

Comment:
this will always create a new reference, you could put this at the root of a file as DefaultJustifyContentFlexAlignment, to save the equality functions some cpu cycles :)

Reply:
great idea! added in https://github.com/concrete-utopia/utopia/pull/3145/commits/bde9df8a6997edf0117cf67ac001bdacef964827